bungiesearch/signals.py

```diff
@@ -17,32 +18,39 @@ def get_signal_processor():
         signal_class = BungieSignalProcessor
     return signal_class()
 
-__items_to_be_indexed__ = defaultdict(list)
```

Comment:
the variable name should not have __ at the end because the pattern `__**__` is reserved for Python implementation methods